src/Microsoft.DocAsCode.Build.Engine/Incrementals/BuildVersionInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.Engine
+{
+    public class BuildVersionInfo
+    {
+        /// <summary>
+        /// The name of version.
+        /// </summary>
+        public string Version { get; set; }
```

Comment:
It is document version name, named by @superyyrrzz , and I don't want to create an alias for it.